DQMServices/Components/src/DQMDaqInfo.cc

```diff
@@ -11,7 +11,7 @@ void DQMDaqInfo::beginLuminosityBlock(const edm::LuminosityBlock& lumiBlock, con
   
   edm::eventsetup::EventSetupRecordKey recordKey(edm::eventsetup::EventSetupRecordKey::TypeTag::findType("RunInfoRcd"));
   
-  if( nullptr != iSetup.find( recordKey ) ) {
+  if( std::nullopt != iSetup.find( recordKey ) ) {
```

Comment:
Should this also be changed to use `tryToGet`?

Reply:
Yes...I missed that one.  Will change.